mevislab.github.io/content/tutorials/image_processing/image_processing6.md

```diff
@@ -4,194 +4,174 @@ date: 2025-05-05
 
-Shows the names (labels) next to each contour in the image.
-* **SoView2DCSOExtensibleEditor**
+👉 After connecting, click on each of these items to visualize the corresponding data in the viewer, as shown in the figures below.
 
+![RTPLAN View2D](/images/tutorials/image_processing/RTPLAN.png "RTPLAN View2D")
+![RTSTRUCT View2D](/images/tutorials/image_processing/RTSTRUCT.png "RTSTRUCT View2D")
 
-Allows you to view and interact with the contours in a 2D image viewer.
-* **View2D**
+![CT View2D](/images/tutorials/image_processing/ctct.png "CT View2D")
+![RTDOSE View2D](/images/tutorials/image_processing/RTDOSE.png "RTDOSE View2D")
 
+We select the CT 512×512×272×1 because it’s the foundation for viewing and aligning all other radiotherapy data.
 
-Displays 2D medical images (like CT scans) with overlays such as contours and dose.
 
-![Connection and the output of View2D](/images/tutorials/image_processing/ConnectionandtheoutputofView2D.png "Connection and the output of View2D")
+We now want to view the CT images and the RTSTRUCT files together. The module `DicomImport` only allows to select one single object. In order to select more than one object, we use a `DicomImportExtraOutput` module. Select the CT series in the `DicomImport` module and the RTSTRUCT in the `DicomImportExtraOutput` module.
 
+You have to select the right index for the RTSTRUCT. In our example it is index 2.
+Now add `DicomImportExtraOutput`module as showen in the below figure :
 
-👉As we can see here, there are no contours displayed in the image yet, so we need to add them:
+![Selecting CT and RTSTRUCT in DicomImport](/images/tutorials/image_processing/SelectingC.png "Selecting CT and RTSTRUCT in DicomImport")
 
 
-From `DicomImportExtraOutput` change using the arrow to 2/4 to show the contour as it show in the figure:
+### **Visualize RTSTRUCTs as colored CSOs:**
 
-![panel DicomImportExtraOutput to show the contourD](/images/tutorials/image_processing/panelDicomImportExtraOutputtoshowthecontour.png "panel DicomImportExtraOutput to show the contour")
+Now we need  `ExtractRTStruct`module to converts RTSTRUCT data into CSOs (Contour Segmentation Objects). CSOs allow MeVisLab to display and interact with the contours on the CT scan.
 
+Then coonect it with the `DicomImportExtraOutput` as shown in the figure :
 
-Now open the View2D panel, its will show the contour with numbers as it’s shown in the below figure :
+![ExtractRTStruct vonverter](/images/tutorials/image_processing/converter.png "ExtractRTStruct converter")
 
-![panel of View2D with contouring and numbers](/images/tutorials/image_processing/panelofView2Dwithcontouringandnumbers.png "panel of View2D with contouring and numbers")
 
+Add `SoView2DCSOExtensibleEdito ` to enable visualization and interaction with the CSOs in the 2D viewer, allowing you to edit and work with the contours directly. Connect it with `View2D`D and `ExtractRTStruct`, as shown, where `View2D` displays the CT scan with the contours.
```

Comment:
Add _an_ ...
typo in the module name
with the `View2D` and `ExtractRTStruct` modules